crates/fuzzing/src/generators/api.rs

```diff
@@ -165,7 +167,7 @@ impl Arbitrary for ApiCalls {
                     <Swarm as Arbitrary>::size_hint(depth),
                     // `arbitrary_config` uses two bools when
                     // `swarm.config_debug_info` is true.
-                    <(bool, bool) as Arbitrary>::size_hint(depth),
+                    <(bool, bool, bool) as Arbitrary>::size_hint(depth),
```

Comment:
This should be four bools because it uses two per config (one for it the swarm is enabled, another for if it actually sets the config flag).

Also the comment above should be updated.